common/components/deploy_model/spec.yaml

```diff
@@ -0,0 +1,77 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: deploy_model
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: Deploy model
+description:
+  Deploy a model to a workspace. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
```

Comment:
Should we add support for custom models as well? It would require scoring script and env at runtime. How will we capture those details from user?

Reply:
Yes ! Initial plan was for ml flow only, so not added support for that.
Should we add it in another pr as next features?

Reply:
Yes, next PR sounds okay to me if its a lot of effort